cypress/support/elements/clue/ResourcesPanel.js

```diff
@@ -1,4 +1,35 @@
 class LeftNav{
```

Comment:
May be a good idea to change the class name to ResourcePanel instead of LeftNav

Reply:
Done